config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  resources :releases
+  resources :releases do
+    resources :reviews
+  end
```

Comment:
please use the "only" syntax with this, to only open up paths for index, new and create (or any other actions you have in your releases controller.

Similarly, do the same for reviews!